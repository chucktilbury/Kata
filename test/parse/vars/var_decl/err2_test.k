;-------------------------------------------------------------------------------
; Show that a missing type name creates the correct error
;-------------------------------------------------------------------------------

var 123.123 some_name
