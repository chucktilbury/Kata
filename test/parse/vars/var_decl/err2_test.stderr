Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err2_test.k: 5: 12: expected a variable declaration but got literal float
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err2_test.k: 5: 12: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_decl/err2_test.k: 5: 12: module cannot be empty
